### PR TITLE
Implements API on my-tickets/list route

### LINKED ISSUE #693: My tickets page does not show tickets, but shows lots of events
E.g. https://nextgen.eventyay.com/my-tickets

Shows events, that were never booked and simply link to the event page instead of showing tickets of the user. Seems like everything is wrong on this page.

Wrong:
screenshot from 2017-08-13 16-40-03

Correct: There are currently no upcoming events booked.
screenshot from 2017-08-13 16-42-22

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Implements API on my-tickets/list route & adds attendee model

Fixes #693

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-737-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@fossasia/open-event-frontend Please review. There will be some console errors, due to event not getting loaded in the `include` clause of the query. The hasMany query does not support the including relationships.

@niranjan94 Are we going to override the ember-data-has-many-query?

**Comment by niranjan94** [comment]
Deployment link: https://pr-737-open-event-frontend-fossasia.surge.sh

File: app/routes/my-tickets/list.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            op   : 'lt',
+            val  : moment().toISOString()
+          }
+        }
+      ];
+    } else {
+      filterOptions = [];
+    }
+    return this.get('authManager.currentUser').query('attendees', {
+      include : 'ticket,event,order',
```
Comment (CosmicCoder96) [inline app/routes/my-tickets/list.js]: @geekyd Included works with ember-has many query right ?

Comment (CosmicCoder96) [inline app/routes/my-tickets/list.js]: @geekyd `include` works with ember has many query right ?

Comment (geekyd) [inline app/routes/my-tickets/list.js]: Not really, if you check the logs, there is a separate call for events. Even for my sessions, there is a separate event call.

File: app/models/attendee.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  firstname   : attr('string'),
+  lastname    : attr('string'),
+  isCheckedIn : attr('boolean'),
+  state       : attr('string'),
+  address     : attr('string'),
+  pdfUrl      : attr('string'),
+  country     : attr('string'),
+  email       : attr('string'),
+
+  event  : belongsTo('event'),
```
Comment (CosmicCoder96) [inline app/models/attendee.js]: @geekyd Add the `attendee` relationship to `event` and `ticket` model too

File: app/models/user.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ export default ModelBase.extend({
   events             : hasMany('event'),
   sessions           : hasMany('session'),
   invoice            : hasMany('event-invoice'),
+  attendees          : hasMany('attendee'),
```
Comment (CosmicCoder96) [inline app/models/user.js]: Relationship of user in attendee model is missing.

Comment (geekyd) [inline app/models/user.js]: Done!!

**Comment by niranjan94** [comment]
Deployment link: https://pr-737-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-737-open-event-frontend-fossasia.surge.sh

File: tests/unit/models/attendee-test.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('attendee', 'Unit | Model | attendee', {
+  // Specify the other units that are required for this test.
+  needs: []
```
Comment (CosmicCoder96) [inline tests/unit/models/attendee-test.js]: @geekyd Please add the relavant needed models here

**Comment by niranjan94** [comment]
Deployment link: https://pr-737-open-event-frontend-fossasia.surge.sh

File: tests/unit/models/attendee-test.js (hunk 4 above)
Comment (geekyd) [inline tests/unit/models/attendee-test.js]: Done!

**Comment by niranjan94** [comment]
Deployment link: https://pr-737-open-event-frontend-fossasia.surge.sh

**Pull request merged**